docs/manage/connections-integrations/webhook-connections/jira-service-desk.md

```diff
@@ -39,34 +39,25 @@ You need the **Manage connections** [role capability](/docs/manage/users-roles
 
 1. Go to **Manage Data \> Monitoring \> Connections**.
```

Comment:
```suggestion
1. Go to **Manage Data** > **Monitoring** > Connections**.
```
I believe we should just bold UI elements and not the arrows (direction)

Reply:
Yup - this is a long-term project -- a leftover from the initial HTML-to-md conversion.